src/main/java/randoop/operation/TypedClassOperation.java

```diff
@@ -68,12 +70,11 @@ public int hashCode() {
    */
   @Override
   public int compareTo(TypedOperation op) {
-    int result = 0;
```

Comment:
It's confusing that `compareTo` is defined in TypedClassOperation, in TypedOperation, and in TypedTermOperation, and all three implementations have hard-coded knowledge of the complete type hierarchy.
The implementations will be hard to keep consistent.

Please move all the logic into the superclass, TypedOperation.  (It can do run-time tests and casts as needed.)  Remove the definitions of `compareTo` from TypedClassOperation and TypedTermOperation.